src/implementations/twig/components/carousel/carousel.html.twig

```diff
@@ -7,7 +7,8 @@
     - "sr_previous" (string) (default: 'Previous slides') screen reader label for previous button
     - "sr_next" (string) (default: 'Next slides') screen reader label for next button
     - "sr_navigation" (string) (default: 'Go to slide %d') screen reader label for navigation buttons
-    - "sr_autoplay" (string) (default: 'Carousel auto play') screen reader label for autoplay button
+    - "sr_play" (string) (default: 'Play carousel') screen reader label for the play button
```

Comment:
we could add a backward compatibility check, to use the value of the previous "sr_autoplay" by default
similar to what was done here https://github.com/ec-europa/europa-component-library/blob/v3.4.0-dev/src/implementations/twig/components/content-item/content-item.html.twig#L42